client/src/components/PosterList.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Poster from './Poster'
+
+PosterList.propTypes = {
+  list: PropTypes.array,
+}
+
+export default function PosterList({ list }) {
+  return (
+    <StyledPosterList>
+      {list.map(({ id, poster_path, name }) => {
+        return (
+          <li key={id}>
+            <Poster
+              path={
+                poster_path && `https://image.tmdb.org/t/p/w300/${poster_path}`
+              }
+              alt={`Poster von ${name}`}
+            />
+          </li>
+        )
+      })}
```

Comment:
Again, personal preference: I find it a bit cleaner when there is only one return statement.
```suggestion
      {list.map(({ id, poster_path, name }) => (
          <li key={id}>
            <Poster
              path={
                poster_path && `https://image.tmdb.org/t/p/w300/${poster_path}`
              }
              alt={`Poster von ${name}`}
            />
          </li>
        )
      )}
```

Reply:
Good point, looks cleaner to me too.